src/lib.rs

```diff
@@ -1062,6 +1077,42 @@ where
         self.oauth2_client.exchange_code(code)
     }
 
+    ///
+    /// Creates a request builder for exchanging a device code for an access token.
+    ///
+    /// See https://tools.ietf.org/html/rfc8628#section-3.4
+    ///
+    pub fn exchange_device_code(
+        &self,
+    ) -> Result<DeviceAuthorizationRequest<TE>, ConfigurationError> {
+        let request = self.oauth2_client.exchange_device_code();
+        if self.use_openid_scope {
+            Ok(request?.add_scope(Scope::new(OPENID_SCOPE.to_string())))
+        } else {
+            request
+        }
+    }
+
+    ///
+    /// Creates a request builder for exchanging a device code for an access token.
+    /// This method is similar to [`exchange_device_code`][Client::exchange_device_code] but
```

Comment:
I'm confused about this comment. `exchange_device_access_code` and `exchange_device_token` are used sequentially for distinct purposes within the device flow; one isn't a more flexible version of the other, and both should allow setting any necessary parameters via their respective request builders.